packages/workers-shared/package.json

```diff
@@ -23,7 +23,7 @@
 	],
 	"scripts": {
 		"build": "pnpm run clean && pnpm run bundle:asset-server:prod",
-		"bundle:asset-server": "esbuild asset-server-worker/src/index.ts --format=esm --bundle --outfile=dist/asset-server-worker.mjs --sourcemap=external",
+		"bundle:asset-server": "esbuild asset-server-worker/src/index.ts --format=esm --bundle --outfile=dist/asset-server-worker.mjs --sourcemap=external --external:cloudflare:*",
```

Comment:
I wonder if we could use `wrangler deploy --dry-run --outdir=dist/asset-worker.mjs` to do this so that we don't have to keep the esbuild settings in sync?